pages/task-forces.md

```diff
@@ -1,11 +0,0 @@
----
-layout: page-fullwidth
-title: "Task Forces"
-permalink: /task-forces/
-excerpt: Task Forces solve problems, explore new ideas, and support the growth of the HPC Carpentry community.
----
-
-HPC Carpentry is trying to help drive the community through the use of *Task Forces*
```

Comment:
Aren't we doing this any more?

Reply:
Arguable, but I suppose we are. Restored by 08cb128b.